docs/functions/text/encoding/json-escaped-to-text.md

```
@@ -0,0 +1,50 @@
+parent: "Encoding functions"
+grand_parent: "Text functions"
+nav_order: 20
+has_toc: false
+permalink: /functions/text/encoding/json-escaped-to-text/
+tags:
+  - functions
+  - text/encoding
+generated: true
+---
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)